### ISSUE TITLE
Glitch with certain items

### ISSUE THREAD

**Issue by DMellswere** [Sigafoos/journal-links#13]
Foundry 8.6
Latest version of this module
Latest version of PF1e

While this journal is active, when I try to create an item or edit it, the text box for the name will allow me to initially enter a name. If I then tab to the next text box, or hit enter, every subsequent text box in that item page will auto delete text as it is entered, resulting in a blank line with a blinking cursor.

Like wise, in details, it does the same thing so that formulae cannot be entered. It will also prevent me from clicking the description tab, at all, once I'm on the details tab and this bug comes up. 

In short, it is impossible to create items within the PF1e system, while this module is active.
